public/app/plugins/datasource/tempo/mockJsonResponse.json

```diff
@@ -3,12 +3,42 @@
     {
       "resource": {
         "attributes": [
-          { "key": "service.name", "value": { "stringValue": "db" } },
-          { "key": "job", "value": { "stringValue": "tns/db" } },
-          { "key": "opencensus.exporterversion", "value": { "stringValue": "Jaeger-Go-2.22.1" } },
-          { "key": "host.name", "value": { "stringValue": "63d16772b4a2" } },
-          { "key": "ip", "value": { "stringValue": "172.24.0.2" } },
-          { "key": "client-uuid", "value": { "stringValue": "39fb01637a579639" } }
+          {
+            "key": "service.name",
+            "value": {
+              "stringValue": "db"
+            }
+          },
```

Comment:
Was this an auto lint format? I'm simply not sure what changed :D

Reply:
Yeah, that was prettier